docs/src/content/docs/guides/customization.mdx

```diff
@@ -0,0 +1,411 @@
+---
+title: Customizing Starlight
+description: Learn how to make your Starlight site your own with custom styles, fonts, and more.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import FileTree from '../../../components/file-tree.astro';
+
+Starlight provides sensible default styling and features, so you can get started quickly with no configuration needed.
+When you want to start customizing the look and feel of your Starlight site, this guide has you covered.
+
+## Add your logo
+
+Adding a custom logo to the site header is a quick way to add your individual branding to a Starlight site.
+
+1. Add your logo image file to the `src/assets/` directory:
+
+   <FileTree>
+
+   - src/
+     - assets/
+       - **my-logo.svg**
+     - content/
+   - astro.config.mjs
+
+   </FileTree>
+
+2. Add the path to your logo as Starlight’s `logo.src` option in `astro.config.mjs`:
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+     integrations: [
+       starlight({
+         title: 'Docs With My Logo',
+         logo: {
+           src: '/src/assets/my-logo.svg',
+         },
+       }),
+     ],
+   });
+   ```
+
+By default, the logo will be displayed alongside your site’s `title`.
+If your logo image already includes the site title, you can visually hide the title text by setting the `replacesTitle` option.
+The `title` text will still be included for screen readers, so that the header remains accessible.
+
+```js
+starlight({
+  title: 'Docs With My Logo',
+  logo: {
+    src: '/src/assets/my-logo.svg',
+    replacesTitle: true,
+  },
+}),
+```
+
+### Light and dark logo variants
+
+You can display different versions of your logo in light and dark mode.
```

Comment:
Considering we're referring to both themes (light and dark), "mode" should be changed to "modes" to be in agreement. The same way the heading is (light and dark variant**s**).
```suggestion
You can display different versions of your logo in light and dark modes.
```